### PR TITLE
create how-it-works page with vid and steps

### PR DESCRIPTION
**Pull request by cwguan**
Sorry for the super large PR (adding 5 SVG illustrations added a lot of lines too ): ) 

https://trello.com/c/8zx4F23U/145-how-it-works-page-video

### CODE REVIEW

File: app/javascript/packs/pages/HowItWorksPage.jsx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    fontSize: "1.6rem",
+    fontWeight: "bold",
+    lineHeight: "50px",
+  },
+}));
+
+const HowItWorks = () => {
+  const classes = useStyles();
+
+  const renderStep = (number, header, content, customContent) => {
```
Comment (tuntisz) [inline app/javascript/packs/pages/HowItWorksPage.jsx:65]: Love the thinking here!

File: app/javascript/packs/pages/HowItWorksPage.jsx [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+        in your community can find it. (Don't worry, your contact information
+        stays private.)
+      </p>
+      <Typography component="p" variant="caption" align="center">
+        This includes nurses, doctors, EMTs, admins, support staff, cleaning
+        staff, and anyone else who is helping during this crisis.
+      </Typography>
+    </>
+  );
+  let content = {
```
Comment (tuntisz) [inline app/javascript/packs/pages/HowItWorksPage.jsx]: Very minor: This can be a constant, I'm indifferent as to whether or not you make the change :)

**Review by tuntisz — APPROVED** [review]
Good stuff! :clinking_glasses:

File: app/javascript/packs/pages/HowItWorksPage.jsx (hunk 2 above)
Comment (cwguan) [reply app/javascript/packs/pages/HowItWorksPage.jsx]: totally agree, updated!

**Comment by cantino** [comment]
Since @tuntisz approved this, feel free to merge it @cwguan!

**Pull request merged**